src/components/section-heading/SectionHeading.tsx

```diff
@@ -0,0 +1,38 @@
+import {StyleSheet} from '@atb/theme';
+import React from 'react';
+import {View} from 'react-native';
+import {ThemeText} from '../text';
+import {ContrastColor, StaticColor, TextColor} from '@atb/theme/colors';
+
+type SectionHeaderProps = {
+  text: string;
+  color?: TextColor | StaticColor | ContrastColor;
+  accessibilityLabel?: string;
+};
+
+export function SectionHeading({
```

Comment:
Is `SectionHeading` the right name? Won't this also be used above other components in the app which isn't `Section`? Added a picture from the front page:
![Screenshot 2023-12-07 at 15 07 46](https://github.com/AtB-AS/mittatb-app/assets/675421/8f879fb6-504f-461d-b514-d04e3ab59ed3)

Reply:
I agree, it's not strictly related to Sections, although it will be right next to one 95% of the time. We could call it something like "ContentHeading". I'm not sure what I prefer.

Ironically, the text you're referring to is also called [SectionHeading](https://github.com/AtB-AS/mittatb-app/blob/master/src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/SectionHeading.tsx). I wanted to replace it in this PR, but agreed with Christian to come back to it, once he had time to look into the figma part of it.

Reply:
Ok, updated now 👍